### PR TITLE
10-11-product-update

### PR DESCRIPTION
**Pull request by MartaD**

### CODE REVIEW

**Review by chrishumanitec — APPROVED** [review]
LGTM

File: docs/change-log.md [hunk 1]
```diff
@@ -2,11 +2,22 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update November 11th, 2022
+
+- **New:** Added autocompletion of [placeholders](https://docs.humanitec.com/reference/concepts/app-config/placeholders) in `labels`, `variables` and `files` properties of a workload in [Deployment Sets](sets-and-deltas.md#deployment-sets).
```
Comment (sujaya-sys) [inline docs/change-log.md:7]: I think we typically capitalise "Workload"

File: docs/change-log.md [hunk 2]
```diff
@@ -2,11 +2,22 @@
...
+
+- **Improved:** Performance of the `All deployment versions` functionality in the [App Details Screen](https://docs.humanitec.com/reference/user-interface/app-details-screen#overview) for applications with a large number of past deployments.
+ 
+- **Fixed:** Image version selector no longer requests all versions even if `show archived` option wasn't checked. It caused performance issues for images with a large number of archived versions.
+
+- **Fixed:** Long [Deploy comments](https://docs.humanitec.com/getting-started/create-sample-app#deploy-the-application) no longer affect accessibility and alignment of action buttons in active deployment cards on the [App Details Screen](https://docs.humanitec.com/reference/user-interface/app-details-screen#overview).
+
+- **Fixed:** An issue related to failing diff of a past deployment after adding a shared resource.
```
Comment (sujaya-sys) [inline docs/change-log.md:15]: We could link to the new shared resources documentation here: https://docs.humanitec.com/using-humanitec/work-with-apps/shared-resources as well as diff deployment: https://docs.humanitec.com/using-humanitec/work-with-environments/deploy-to-environments/diff-deployments

File: docs/change-log.md [hunk 3]
```diff
@@ -2,11 +2,22 @@
...
 
+## Product Update November 11th, 2022
+
+- **New:** Added autocompletion of [placeholders](https://docs.humanitec.com/reference/concepts/app-config/placeholders) in `labels`, `variables` and `files` properties of a workload in [Deployment Sets](sets-and-deltas.md#deployment-sets).
+
+- **Improved:** Performance of the `All deployment versions` functionality in the [App Details Screen](https://docs.humanitec.com/reference/user-interface/app-details-screen#overview) for applications with a large number of past deployments.
+ 
+- **Fixed:** Image version selector no longer requests all versions even if `show archived` option wasn't checked. It caused performance issues for images with a large number of archived versions.
+
+- **Fixed:** Long [Deploy comments](https://docs.humanitec.com/getting-started/create-sample-app#deploy-the-application) no longer affect accessibility and alignment of action buttons in active deployment cards on the [App Details Screen](https://docs.humanitec.com/reference/user-interface/app-details-screen#overview).
```
Comment (sujaya-sys) [inline docs/change-log.md:13]: Not sure if we want to link to a getting started guide here, might be a bit confusing to be dropped into the middle of a tutorial

File: docs/change-log.md [hunk 4]
```diff
@@ -2,11 +2,22 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update November 11th, 2022
+
+- **New:** Added autocompletion of [placeholders](https://docs.humanitec.com/reference/concepts/app-config/placeholders) in `labels`, `variables` and `files` properties of a workload in [Deployment Sets](sets-and-deltas.md#deployment-sets).
+
+- **Improved:** Performance of the `All deployment versions` functionality in the [App Details Screen](https://docs.humanitec.com/reference/user-interface/app-details-screen#overview) for applications with a large number of past deployments.
```
Comment (sujaya-sys) [inline docs/change-log.md:9]: I know we're not consistent with this but I guess we could start capitalising "Applications" as well

File: docs/change-log.md (hunk 1 above)
Comment (MartaD) [inline docs/change-log.md:7]:
```suggestion
- **New:** Added autocompletion of [placeholders](https://docs.humanitec.com/reference/concepts/app-config/placeholders) in `labels`, `variables` and `files` properties of a Workload in [Deployment Sets](sets-and-deltas.md#deployment-sets).
```

File: docs/change-log.md (hunk 3 above)
Comment (MartaD) [inline docs/change-log.md:13]:
```suggestion
- **Fixed:** Long Deploy comments no longer affect accessibility and alignment of action buttons in active deployment cards on the [App Details Screen](https://docs.humanitec.com/reference/user-interface/app-details-screen#overview).
```

File: docs/change-log.md (hunk 2 above)
Comment (MartaD) [inline docs/change-log.md:15]:
```suggestion
- **Fixed:** An issue related to failing [Deployment Diff](https://docs.humanitec.com/using-humanitec/work-with-environments/deploy-to-environments/diff-deployments) of a past deployment after adding a [Shared Resource](https://docs.humanitec.com/using-humanitec/work-with-apps/shared-resources).
```

Comment (MartaD) [reply docs/change-log.md:15]: I've also capitalized Shared Resource and Deployment Diff, let me know if that's fine or not 😊

File: docs/change-log.md (hunk 4 above)
Comment (MartaD) [inline docs/change-log.md:9]:
```suggestion
- **Improved:** Performance of the `All deployment versions` functionality in the [App Details Screen](https://docs.humanitec.com/reference/user-interface/app-details-screen#overview) for Applications with a large number of past deployments.
```

File: docs/change-log.md (hunk 2 above)
Comment (sujaya-sys) [reply docs/change-log.md]: True yeah, we should adapt the Nomenclature of the docs team. Makes sense :-)

**Review by sujaya-sys — APPROVED** [review]
LGTM

**Pull request merged**